packages/py/msgpack/tests/test_msgpack.py

```diff
@@ -0,0 +1,62 @@
+from dataclasses import dataclass
+
+from msgpack_wrap import msgpack_encode, msgpack_decode
+
+expected_array_like = [
+    130, 168, 102, 105, 114, 115, 116, 75,
+    101, 121, 170, 102, 105, 114, 115, 116,
+    86, 97, 108, 117, 101, 169, 115, 101,
+    99, 111, 110, 100, 75, 101, 121, 171,
+    115, 101, 99, 111, 110, 100, 86, 97,
+    108, 117, 101
+]
+
+
+def test_encode_and_decode_object():
+    custom_object = {
+        "firstKey": "firstValue",
+        "secondKey": "secondValue"
+    }
+
+    encoded = msgpack_encode(custom_object)
+    assert encoded == bytes(expected_array_like)
+
+    decoded = msgpack_decode(encoded)
+    assert decoded == custom_object
```

Comment:
encode dictionary as our generic map and encode dataclasses as the object (msgpack map family).